geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionConfig.java

```diff
@@ -2057,6 +2057,8 @@
 
   /**
    * The name of the Diffie-Hellman symmetric algorithm "security-client-dhalgo" property.
+   *
+   * @deprecated since Geode 1.5. Use SSL instead. See {@link #SSL_ENABLED_COMPONENTS_NAME}
```

Comment:
You put this on the security-udp-dhalgo property because the docstring is wrong. That should be deprecated as well but at least the comment should be correct.

Reply:
Good catch! I fixed it.